eng/pipelines/templates/jobs/ci.tests.yml

```diff
@@ -117,9 +117,9 @@ jobs:
           mergeTestResults: true
       - template: /eng/common/pipelines/templates/steps/publish-artifact.yml
         parameters:
-          ArtifactPath: $(Build.ArtifactStagingDirectory)/*.binlog
-          ArtifactName: logs
-          CustomCondition: eq(variables['System.Debug'], 'true')
+          ArtifactPath: $(Build.ArtifactStagingDirectory)/test.binlog
+          ArtifactName: binlog_$(Agent.JobName)
```

Comment:
Thanks. I swear I tested this, but maybe the conditional issue you fixed in the other file (`System.Debug` was documented to be available, but I didn't realize setting it queue time doesn't just update that globally scoped variable) and don't remember any conflicts. Besides, I incorrectly remembered AzDO can't just append to artifacts like GitHub Actions can...and I've been writing a heck of a lot more GitHub Actions the past few years.